usagov/usagov-2021
# USAGOV-1204 Add Back Cancel Button To Search

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1204

## Description

Remove the sass that removed the cancel button from search

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On localhost/agency-index go to the search box. Check that now when you start typing the cancel button is in the search bar and functional. 

Used to look like: 
![Screenshot 2023-11-29 at 8 29 34 AM](https://github.com/usagov/usagov-2021/assets/108532577/e8deaeb2-aa95-48c3-b47d-47386171da30)

Should now look like: 
![Screenshot 2023-11-29 at 8 28 40 AM](https://github.com/usagov/usagov-2021/assets/108532577/8cd229cb-6130-4d43-a326-084b495758f4)


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
